api/aws/tfvars.go

```diff
@@ -18,9 +18,10 @@ type AWSTFVars struct {
 	MachinesSC map[string]*api.Machine `json:"machines_sc" validate:"required,min=1"`
 	MachinesWC map[string]*api.Machine `json:"machines_wc" validate:"required,min=1"`
 
-	PublicIngressCIDRWhitelist []string `json:"public_ingress_cidr_whitelist" validate:"required"`
-	APIServerWhitelist         []string `json:"api_server_whitelist" validate:"required"`
-	NodeportWhitelist          []string `json:"nodeport_whitelist" validate:"required"`
+	PublicIngressCIDRWhitelist []string          `json:"public_ingress_cidr_whitelist" validate:"required"`
+	APIServerWhitelist         []string          `json:"api_server_whitelist" validate:"required"`
+	NodeportWhitelist          []string          `json:"nodeport_whitelist" validate:"required"`
+	ExtraTags                  map[string]string `json:"extra_tags"`
```

Comment:
Is it tested to set this to nothing? It might be that terraform requires this variable, and then this would fail.
In that case I would just initialize it to an empty map in the aws default flavor.